Assets/Scripts/Gameround/GameRound.cs

```diff
@@ -73,12 +99,26 @@ private void CheckIfPlayerOnTreasure()
                 {
                     var treasureControl = treasure.GetComponent<Treasure>();
 
-                    treasureControl.TryOpen(player);
+                    var opened = treasureControl.TryOpen(player);
```

Comment:
i think it might be cleaner to pass the loottable as an argument to the try open list
```suggestion
                    var opened = treasureControl.TryOpen(player, loottable);
```